.changeset/tasty-rats-explain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/integration': minor
+---
+
+This patch brings Harness Code as a valid integration via the ScmIntgration interface. It adds harness code to the relevant static properties ( get integration by name, get integration by type) for plugs to be able to reference the same harness code server
```

Comment:
```suggestion
Added `HarnessIntegration` via the `ScmIntegrations` interface.
```

Reply:
fixed @vinzscam thanks for the review :)